widgetry/src/widgets/compare_times.rs

```diff
@@ -100,9 +100,11 @@ impl CompareTimes {
                 .collect(),
         )
         .evenly_spaced();
-        let y_label = Text::from(format!("{} (minutes)", y_name.as_ref()))
+        let mut y_label = Text::from(format!("{} (minutes)", y_name.as_ref()))
             .render(ctx)
-            .rotate(Angle::degrees(90.0))
+            .rotate(Angle::degrees(90.0));
+        y_label.autocrop_dims = true;
```

Comment:
I understand from a code standpoint why this fixes the problem, but I'm not really sure why the "autocrop" method doesn't do this by default.

Reply:
I had to dig around to remember why we don't always use `autocrop_dims = true`. The problem is with text -- we want to retain the blank space on the left and right sides when laying out text "normally" / not rotated. I know sometimes we add a `Color::CLEAR` rectangle behind things to "force" a proper boundary. If we do this for text, maybe we can get rid of `autocrop_dims` entirely (and act as if it's always true), so then the rotation should just work as expected...